octoprint_mrbeam/iobeam/lid_handler.py

```diff
@@ -734,21 +738,17 @@ def serve_pictures(self, cam, pic_settings=None, cam_params=None, out_pic_size=N
 				missed=missed,
 				increment_pic=True,
 				error=err,
-				extra=analytics
+				extra=self.analytics
 			)
 
-			# upload image to analytics if env is dev
-			if self._plugin.is_dev_env() and self._settings.get(['dev', 'automatic_camera_image_upload'])\
-					and latest is not None \
-					and (loop_counter <= 10 \
-					or  (loop_counter > 10 and loop_counter % 10 == 0)):
-				count_sent_pictures_analytics += 1
-				self._send_last_img_to_analytics(latest, 'dev_auto',
-								 self.last_markers, missed,
-								 min_pix_amount,
-								 analytics,
-								 force_upload=(count_sent_pictures_analytics%10==0),
-								 notify_user=False)
+			# upload image to analytics
+			pic_counter += 1
+			force = pic_counter <= 10 \
+			        or  (pic_counter > 10 and pic_counter % 10 == 0)
+			if self._plugin.is_dev_env() \
+			        and self._settings.get(['dev', 'automatic_camera_image_upload']) \
+			        and force:
```

Comment:
this seems to change the logic:
while force_upload was originally set only every xth added image, now it's never set but only every xth image is added to analytics